src/Parser/DeathKnight/Blood/Modules/Talents/WillOfTheNecropolis.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from "common/format";
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+const MINIMUMABSORTBTHRESHOLD = 0.05;
```

Comment:
looks like an extra T managed to sneak it's way in there :P also please use underscores to split the words in constants.